templates/components/faceted-search/selected-facets.html

```diff
@@ -15,7 +15,7 @@ <h5 class="sidebarBlock-heading">
                         {{#if facet '===' 'rating'}}
                             {{lang 'search.faceted.selected.rating-label' rating=title}}
                         {{else}}
-                            {{ title }}
+                            {{{ title }}}
```

Comment:
I think we should do it across all these files https://github.com/bigcommerce/cornerstone/tree/master/templates/components/faceted-search/facets

Reply:
I have made changes to all applicable files. Please review them.